### PR TITLE
Add start trial documentation

### PR DESCRIPTION
**Pull request by handlers**

### CODE REVIEW

**Comment by mootpt** [comment]
lgtm

File: organizations/trials.html.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Start a Trial"
+---
+
+# Start a trial
+
+Atlas offers organizations 30-day trials for Terraform Enterprise, [Consul Enterprise](https://www.hashicorp.com/consul.html), and Vagrant Enterprise. Note that trials are available for organizations, not users.
```
Comment (KFishner) [inline organizations/trials.html.md]: Terraform Enterprise will go out today so we can link to it

/terraform.html

File: organizations/trials.html.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+To start a trial for your organization:
+
+1. Create an Atlas user account if you don't already have one
```
Comment (KFishner) [inline organizations/trials.html.md]: we might as well link to a /create/new here. It's pretty hidden and I'm not worried about a deluge of new accounts

Comment (KFishner) [inline organizations/trials.html.md]: plus we can link to this and have a user handle everything his or herself

**Comment by handlers** [comment]
Added the links; merging.

**Pull request merged**